BasicSample/app/build.gradle

```diff
@@ -48,29 +48,24 @@ android {
 
 dependencies {
     compile fileTree(include: ['*.jar'], dir: 'libs')
-    compile 'com.android.support:appcompat-v7:' + rootProject.supportLibVersion;
-    compile 'com.android.support:cardview-v7:' + rootProject.supportLibVersion;
-    compile 'com.android.support:recyclerview-v7:' + rootProject.supportLibVersion;
-    compile 'android.arch.lifecycle:extensions:' + rootProject.archLifecycleVersion;
-    compile 'android.arch.persistence.room:runtime:' + rootProject.archRoomVersion;
-    annotationProcessor "android.arch.lifecycle:compiler:" + rootProject.archLifecycleVersion;
-    annotationProcessor "android.arch.persistence.room:compiler:" + rootProject.archRoomVersion;
+    compile 'com.android.support:design:' + rootProject.supportLibVersion
```

Comment:
these are still using `compile` whereas in other project you changed it to `implementation`

Reply:
There are only two projects use new Android plugin: `BasicRxJavaSampleKotlin` & `ithubBrowserSample`. `implementation` should be used only with new Android plugin.
Other projects still use stable plugin, so they should stick with `compile`.